public/components/map_container/mapsFooter.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiPanel } from '@elastic/eui';
+import React, { memo } from 'react';
+import { LngLat } from 'maplibre-gl';
+
+const coordinatesRoundOffDigit = 4;
+interface MapFooterProps {
+  coordinates?: LngLat;
+  zoom: number;
+}
+
+// props are equal only if zoom and lat/lng are equal
+const propsComparator = (prevProps: MapFooterProps, nextProps: MapFooterProps) => {
```

Comment:
We probably can make use of lodash to compare

```
const propsComparator = (prevProps, nextProps) => {
  return _.isEqual(prevProps, nextProps);
};
```
Ref: https://github.com/opensearch-project/dashboards-maps/blob/main/public/components/layer_control_panel/layer_control_panel.tsx#L144-L149

Reply:
Ack